core/src/test/scala/akka/kafka/scaladsl/IntegrationSpec.scala

```diff
@@ -197,9 +208,70 @@ class IntegrationSpec extends TestKit(ActorSystem("IntegrationSpec"))
       Await.result(control.isShutdown, remainingOrDefault)
     }
 
-    // TODO add more tests
+    "consume and commit in batches" in {
```

Comment:
My understanding is that if the offsets are not committed then this verification step 
```
probe
        .request(50)
        .expectNextN((101 to 150).map(_.toString))
```

would fail since messages would be consumed from the beginning (i.e. message "1").